src/chunkserver/copyset_node.cpp

```diff
@@ -168,6 +172,9 @@ int CopysetNode::Init(const CopysetNodeOptions &options) {
     raftNode_ = std::make_shared<RaftNode>(groupId, peerId_);
     concurrentapply_ = options.concurrentapply;
 
+    // init braft lease
+    braft::FLAGS_raft_enable_leader_lease = true;
```

Comment:
A better way is add another option in chunkserver.conf, like `copyset.enable_lease_read`, and parse this option when chunkserver startup, if it's true, then set braft::FLAGS_raft_enable_leader_lease to true.

Reply:
> A better way is add another option in chunkserver.conf, like `copyset.enable_lease_read`, and parse this option when chunkserver startup, if it's true, then set braft::FLAGS_raft_enable_leader_lease to true.

If disable this flag, all requests (including read and write requests) will propose to raft?